sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RequestTimeline.java

```diff
@@ -151,12 +151,15 @@ public String toString() {
         return RntbdObjectMapper.toString(this);
     }
 
-    @JsonPropertyOrder({ "name", "startTimeUTC" })
+    @JsonPropertyOrder({ "name", "startTimeUTC", "durationInMicroSec" })
     public static final class Event {
 
         @JsonIgnore
         private final Duration duration;
 
+        @JsonSerialize(using = ToStringSerializer.class)
```

Comment:
I know that this used to be `ToStringSerializer`. but the type is long and I wonder we should drop the ToStringSerializer.

the difference is how this gets serialized: 5 vs "5"

Reply:
yea, make sense, will update

Reply:
updated